src/include/common/types/value.h

```diff
@@ -3,6 +3,7 @@
 #include "common/api.h"
 #include "common/exception.h"
 #include "common/type_utils.h"
+#include "common/types/blob.h"
```

Comment:
you should probably put this include inside types/types_include.h. So that you don't need to include it here